docs/advanced/pycpp/object.rst

```diff
@@ -13,6 +13,11 @@ Available types include :class:`handle`, :class:`object`, :class:`bool_`,
 :class:`iterable`, :class:`iterator`, :class:`function`, :class:`buffer`,
 :class:`array`, and :class:`array_t`.
 
+.. warning::
+
+    Be sure to review the :ref:`gotchas` before using this heavily in your C++
```

Comment:
Shouldn't this be

```
:ref:`gotchas <pytypes_gotchas>`
```

Reply:
Oops! Yeah, I misspelled the refname. Fixed now.

FWIW ``:ref:`pytypes_gotchas``` worked too:
![image](https://user-images.githubusercontent.com/26719449/89739804-ac87ab80-da51-11ea-89be-7f6b30d2c036.png)